### ISSUE TITLE
ncvis crashes when trying to plot zonal mean data

### ISSUE THREAD

**Issue by whannah1** [SEATStandards/ncvis#9]
I'm calculating EP flux vectors as part of the transformed Eulerian mean budget, which results in data that has dimensions of (time,lev,lat). The resulting file looks like this:

```
> ncdump -h /global/cscratch1/sd/whannah/e3sm_scratch/cori-knl/E3SM.QBO-TEST.F2010.ne30pg2.L72.01/data_remap_180x360_prs/E3SM.QBO-TEST.F2010.ne30pg2.L72.01.eam.h2.tem.0001-01-04-00000.remap_180x360.nc
netcdf E3SM.QBO-TEST.F2010.ne30pg2.L72.01.eam.h2.tem.0001-01-04-00000.remap_180x360 {
dimensions:
	lat = 180 ;
	plev = 37 ;
	time = 4 ;
variables:
	double lat(lat) ;
		lat:_FillValue = NaN ;
		lat:long_name = "Latitude of Grid Cell Centers" ;
		lat:standard_name = "latitude" ;
		lat:units = "degrees_north" ;
		lat:axis = "Y" ;
		lat:valid_min = -90. ;
		lat:valid_max = 90. ;
		lat:bounds = "lat_bnds" ;
	double plev(plev) ;
		plev:_FillValue = NaN ;
	double time(time) ;
		time:_FillValue = NaN ;
		time:long_name = "time" ;
		time:bounds = "time_bnds" ;
		time:units = "days since 0001-01-01" ;
		time:calendar = "noleap" ;
	double dFydy(time, plev, lat) ;
		dFydy:_FillValue = NaN ;
	double dFzdp(time, plev, lat) ;
		dFzdp:_FillValue = NaN ;
	double EP_div(time, plev, lat) ;
		EP_div:_FillValue = NaN ;
}
```

I thought I'd use ncvis to do a quick sanity check, but when I went to select one of the variables it crashed with this message:

```
> /global/homes/p/paullric/ncvis/ncvis /global/cscratch1/sd/whannah/e3sm_scratch/cori-knl/E3SM.QBO-TEST.F2010.ne30pg2.L72.01/data_remap_180x360_prs/E3SM.QBO-TEST.F2010.ne30pg2.L72.01.eam.h2.tem.0001-01-04-00000.remap_180x360.nc
NcVis 2022.08.28 Paul A. Ullrich
EXCEPTION (/global/homes/p/paullric/ncvis/src/wxImagePanel.cpp, Line 1221) Assertion failure
```

**Comment by paullric** [comment]
Could you grant read permissions to that file?

**Comment by whannah1** [comment]
@paullric sorry about that, permissions should be open now. 

I should also mention that this might have to do with the presence of NaNs. I used a centered difference for the derivatives, so the points around the edges were set to numpy's NaN value.

**Comment by paullric** [comment]
Fixed in e676db2411077e23fabd8eadefb03f4932a5432a.  Indeed the issue was no support for NaN fillvalues.
